auditlog_tests/tests.py

```diff
@@ -1273,6 +1273,12 @@ def test_register_from_settings_invalid_settings(self):
             ):
                 self.test_auditlog.register_from_settings()
 
+        with override_settings(AUDITLOG_DISABLE_REMOTE_ADDR="str"):
```

Comment:
You need to move this test in (MiddlewareTest)[https://github.com/jazzband/django-auditlog/blob/5e2daa4c4c17269761fb76391c2cad900d3b4cd8/auditlog_tests/tests.py#L449] because the checking is in middleware init